dc-cudami-server/dc-cudami-server-webapp/src/main/java/de/digitalcollections/cudami/server/controller/legacy/V5MigrationHelper.java

```diff
@@ -0,0 +1,80 @@
+package de.digitalcollections.cudami.server.controller.legacy;
+
+import static de.digitalcollections.model.list.sorting.Direction.ASC;
+import static de.digitalcollections.model.list.sorting.Direction.DESC;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.openjson.JSONArray;
+import com.github.openjson.JSONObject;
+import de.digitalcollections.model.list.paging.PageResponse;
+import de.digitalcollections.model.list.sorting.Direction;
+import de.digitalcollections.model.list.sorting.Order;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class V5MigrationHelper {
+
+  public static List<Order> migrate(List<Order> sortBy) {
+    List<Order> result = null;
+    if (sortBy != null) {
+      result =
+          sortBy.stream()
+              .map(
+                  o -> {
+                    o.setIgnoreCase(true);
+                    return o;
+                  })
+              .collect(Collectors.toList());
+    }
+    return result;
+  }
+
+  public static String migrate(PageResponse<?> pageResponse, ObjectMapper objectMapper)
+      throws JsonProcessingException {
+    if (pageResponse == null) {
+      return null;
+    }
+
+    // For each order of the sorting, we have to re-append the values for ascending and descending
+    JSONObject jsonObject = new JSONObject(objectMapper.writeValueAsString(pageResponse));
+    return migrateToV5(jsonObject, objectMapper);
+  }
+
+  public static String migrateToV5(String unmigratedJson) {
+    String migratedJson = unmigratedJson.replaceAll("executedSearchTerm", "query");
```

Comment:
Sollte das Feld nicht einfach gelöscht werden? Das gab es ja bisher nicht.

Reply:
doch, in SearchPageResponse hat es das Feld gegeben...

Reply:
`query` schon, aber das wird ja bereits in der Zeile drunter belegt.

Reply:
existiert zweimal (in request und in response)